### DISCUSSION TITLE
Getting around 5 seconds delay between clicking and the click registering on LAN

### DISCUSSION THREAD

**Discussion by user529481** [FreeRDP/FreeRDP#9445] [Q&A]
I'm connecting a machine on my local network running debian 12 with gnome, and I'm connecting from mac, with xQuartz. This is the command I use to start my session:

xfreerdp +clipboard /u:dan /v:10.0.0.46 /network:lan

Mouse movement is fairly quick, but clicking and having the click register takes several seconds to register. Does anyone know how I can diagnose this problem?

**Comment by akallabeth** [comment]
@user293811 sounds like some kind of networking issue? but you never know until you try, so:

1. mouse movement is your local mouse, so no delay expected
2. you may notice delays similar to that of a click if you e.g. move over a icon (where there is actually network traffic involved)
3. you may try different clients to connect (e.g. royaltsx, thincast client, microsoft rdp, ...) and check if they have a similar delay
